redis/src/main/scala/zio/redis/options/Strings.scala

```diff
@@ -64,4 +64,53 @@ trait Strings {
   }
 
   type KeepTtl = KeepTtl.type
+
+  sealed trait StrAlgoCommand
+
+  object StrAlgoCommand {
+    case class LongestCommonSubsequence(
+      lcsType: LcsType,
+      input1: String,
+      input2: String,
+      len: Option[QueryType] = None,
+      minMatchLen: Option[MinMatchLen] = None,
+      withMatchLen: Option[WithMatchLen] = None
+    ) extends StrAlgoCommand
+
+    sealed trait LcsType { self =>
```

Comment:
Maybe move these out of the `StrAlgoCommand` object and just leave `LongestCommonSubsequence` in it.